packages/twenty-server/src/core/open-api/utils/compute-schema-tags.utils.ts

```diff
@@ -0,0 +1,17 @@
+import { ObjectMetadataEntity } from 'src/metadata/object-metadata/object-metadata.entity';
+import { capitalize } from 'src/utils/capitalize';
+
+export const computeSchemaTags = (
+  items: ObjectMetadataEntity[],
+): { name: string; description: string }[] => {
+  const results = items.map((item) => {
+    return {
+      name: item.namePlural,
+      description: `Object \`${capitalize(item.namePlural)}\``,
+    };
+  });
+
+  results.push({ name: 'Others', description: 'Other routes' });
```

Comment:
@martmull I think it's more readable to declare the static part first

```suggestion
  const results: { name: string; description: string; }[] = [{ name: 'Others', description: 'Other routes' }];
  
  for (const item of items) {
    results.push({
      name: item.namePlural,
      description: `Object \`${capitalize(item.namePlural)}\``,
    });
  }
```